src/components/dex/MarketPairChart.vue

```diff
@@ -14,32 +14,32 @@
         <div class="day-values">
           <div class="row">
             <div class="label">VOL.</div>
-            <div class="value">{{ tickerData.quoteVolume }}</div>
+            <div class="value">{{ tickerData[currentMarket].quoteVolume }}</div>
```

Comment:
If `tickerData` or `tickerData[currentMarket]` are undefined, this won't render. We need to take that into account. Also, further in this component, there are similar situations.